perf/latency/limbo/gen-database.py

```diff
@@ -24,10 +25,13 @@
         address TEXT,
         city TEXT,
         state TEXT,
-        zipcode TEXT
+        zipcode TEXT,
+        i TEXT
```

Comment:
Why do we need tenant index in the database? Every tenant should use their own separate database, which makes this redundant.

Reply:
I am putting this to assert the read coming from the proper database.